libretro-common/include/libretro.h

```diff
@@ -976,6 +976,32 @@ struct retro_hw_render_context_negotiation_interface
                                             * so it will be used after SET_HW_RENDER, but before the context_reset callback.
                                             */
 
+#define RETRO_SERIALIZATION_QUIRK_INCOMPLETE (1 << 0)
+   /* Serialized state is incomplete in some way. Set if serialization is
+    * usable in typical end-user cases but should not be relied upon to
+    * implement frame-sensitive frontend features such as netplay or
+    * rerecording. */
```

Comment:
I guess I was confused because the comments for environments seemed consistently below. I'll do my best to be consistent with the inconsistency :)